shopfloor/services/service.py

```diff
@@ -23,6 +24,51 @@ class BaseShopfloorService(AbstractComponent):
     _actions_collection_name = "shopfloor.action"
     _expose_model = None
 
+    def dispatch(self, method_name, _id=None, params=None):
+        try:
+            result = super().dispatch(method_name, _id=_id, params=params)
+        except Exception as err:
+            self.env.cr.rollback()
+            with registry(self.env.cr.dbname).cursor() as cr:
+                env = self.env(cr=cr)
+                self._log_call_in_db(env, _id, params, error=err)
+            raise
+        self._log_call_in_db(self.env, _id, params, result=result)
```

Comment:
isn't this too much? Can't we make it only for specific services? I'd add a class flag to do it only where we need (eg: no reason to track logs for loading menu or profiles).

Reply:
Are you sure there are no reason? Knowing the user didn't refresh the menu after a manager changed the configuration is something I'd like to know the day a weird bug happens (even if today it shouldn't be an issue).

Reply:
That being said, you are right it's a good idea to have a flag, I'll add it